lms/templates/provider/authorize.html

```diff
@@ -1,59 +1,66 @@
-{% extends "main_django.html" %}
+## mako
```

Comment:
I don't see a straightforward way to make any of these templates pick up the non-courseware version of the static assets. I think that will just have to be okay.